recipes/default.rb

```diff
@@ -3,7 +3,13 @@
 
 venv = node['mimic']['virtualenv']
 
-%w(libffi6 libffi-dev).each do |pkg|
+if node['lsb']['release'] < '20'
+  packages = %w(libffi6 libffi-dev)
+else
+  packages = %w(libffi7 libffi-dev)
+end
+
+packages.each do |pkg|
```

Comment:
Oh, woops. My previous comment wasn't quite accurate, but you can pass multiple packages to the `package` resource. No need for a loop. It should also do the install in a single transaction, making it faster than doing one at a time.